requirements_docs.txt

```diff
@@ -1,33 +1,32 @@
-cmocean
-colorcet
-imageio-ffmpeg
+Sphinx==4.4.0
+cmocean==2.0
+colorcet==3.0.0
+https://github.com/numpy/numpydoc/archive/main.zip  # need latest
```

Comment:
Can we request a release?

Reply:
Boom: https://github.com/numpy/numpydoc/issues/355